actionpack/lib/action_dispatch/routing/url_for.rb

```diff
@@ -147,6 +149,24 @@ def url_options
       #    # => 'http://somehost.org/myapp/tasks/testing'
       #    url_for controller: 'tasks', action: 'testing', host: 'somehost.org', script_name: "/myapp", only_path: true
       #    # => '/myapp/tasks/testing'
+      #
+      # Missing Routes keys may be filled in from the current request's parameters
+      # (e.g. +:controller+, +:action+, +:id+ and any other parameters that are
+      # placed in the path). This allows one to have the following code:
+      #
+      #   class UsersController < Rails::Application
+      #     # GET /users/1
+      #     def show
+      #       url_for(only_path: true)                        # => '/users/1'
+      #       url_for(only_path: true, action: 'edit')        # => '/users/1/edit'
+      #       url_for(only_path: true, action: 'edit', id: 2) # => '/users/2/edit'
+      #     end
+      #   end
+      #
+      # Notice that no parameter +:id+ was provided to the first +url_for+ call
+      # and the helper used the one from the Route’s path. Any path parameter
```

Comment:
Ditto regarding capitalization of "route". Also, could you use a regular apostrophe (') please ?